src/SSHDebugPS/UI/ContainerPickerDialogWindow.xaml

```diff
@@ -30,12 +30,12 @@
         <Style x:Key="GlyphPathStyle"
                TargetType="{x:Type Path}">
             <Setter Property="Stroke"
-                            Value="{DynamicResource {x:Static platformui:TreeViewColors.GlyphBrushKey}}" />
+                            Value="#717171" /> <!-- GlyphBrushKey = Glyph = 717171-->
```

Comment:
Ideally we use system colors i.e. something from this enumeration: https://docs.microsoft.com/en-us/dotnet/api/system.windows.systemcolors?view=net-5.0